src/types/message.ts

```diff
@@ -110,6 +88,34 @@ function isStopMessage(message: Message): boolean {
   return message.type === 'stop';
 }
 
+export interface GeofenceMessage extends MessageBase {
+  type: 'geofence';
+
+  /**
+    * Geofencing coordinates in the form of a polygon.
+    */
+  geofence: {
+    /**
+     * geofence.keepOut and geofence.keepIn are defined as an array of tuples( [[number, number]] ).
+     * Each tuple consists of [ [latitude, longitude] ].
+     */
```

Comment:
lets change the description to something like this `Array of tuples that represent [latitude, longitude]`, and put
the description on top of keepOut and keepIn. message me if you need more clarification